src/PostLM.ts

```diff
@@ -19,12 +19,29 @@ export class PostLM {
       const totalPriceInput = document.evaluate("div[label/span[text()='Total price']]/div/div/input", form, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue as HTMLInputElement;
 
       const displayElement = createTextSpan('-- ea', this.tag);
-      totalPriceInput.parentNode!.insertBefore(displayElement, totalPriceInput);
+        totalPriceInput.parentNode!.insertBefore(displayElement, totalPriceInput);
 
+        const type = document.evaluate(
+            "div[label/span[text()='Type']]/div/div",
+            form, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null
+        ).singleNodeValue as HTMLElement;
+        
       const calculatePricePerUnit = () => {
         const amount = parseInt(amountInput.value);
-        const total = parseInt(totalPriceInput.value);
-        displayElement.textContent = `${toFixed(total / amount, 1)} ea`;
+          const total = parseInt(totalPriceInput.value);
+          const postingFee = (document.evaluate("div[label/span[text()='Fees']]/div/div/div/span", form, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue as HTMLSpanElement).innerText;
```

Comment:
selector didn't work because this field is embedded all the way in tags that are used elsewhere (e.g. CX attributes). i'm open to any other suggestion